src/parse/ris_live.go

```diff
@@ -93,23 +94,28 @@ func ParseRisLiveData(msgChannel chan common.BGPMessage) error {
 	fmt.Println("made connection")
 
 	/* Subscribe */
-	subscription1 := RisMessage{"ris_subscribe", &RisMessageData{"", "0.0.0.0/0"}}
-	//TODO: connect subscription to config file
-	//		keep an array of subscriptions
+	subscriptionPrefixes := strings.Split(configPrefixes, ",")
+
+	if len(subscriptionPrefixes) == 0 {
```

Comment:
I'd say: if there's nothing in the config, rather than subscribe to everything, you could exit and log an error stating that the config is necessary